pages.fr/common/git-stash.md

```diff
@@ -0,0 +1,36 @@
+# git stash
+
+> Mettre les changements locaux de Git dans un endroit temporaire (Le stash).
+> Plus d'informations: <https://git-scm.com/docs/git-stash>.
+
+- Mettre les changements courants (sauf ceux provenant de fichiers qui ne sont pas suivi) dans un *stash*:
+
+`git stash [push -m {{optional_stash_message}}]`
```

Comment:
Is this example parameter already translated? I've quickly thrown it into a translator and it seems to be already translated but also it looks exactly like English so just making sure.

Reply:
Oops, my bad I completely missed the parameter. It is indeed still English.